src/main/php/websocket/Listener.class.php

```diff
@@ -23,7 +23,9 @@ public abstract function message($connection, $message);
    * Closes connection
    *
    * @param  websocket.protocol.Connection $connection
+   * @param  int $code
+   * @param  string $reason
    * @return void
    */
-  public function close($connection) { /* NOOP */ }
+  public function close($connection, $code, $reason) { /* NOOP */ }
```

Comment:
This is a BC break - all `Listener` implementations with a *close()* method need to be adapted!

Reply:
See thekid/crews@3f47d65